### DISCUSSION TITLE
Nearly over 40% partials of 680T chia plots is stale on Space Pool for over 1 month

### DISCUSSION THREAD

**Discussion by squallsdjl** [Chia-Network/chia-blockchain#8580] [Support]
Hi guys, I am a rookie on CLI and I run 8 servers at same place. I just found nearly over 40% partials of 680T chia plots is stale on Space Pool for over 1 month....god...

For the first time I just thought I am plotting as farming, this might have some side effect, etc. But now I have plot them full and just stay farming. Space Pool still shows that data. I just don't know what to do then. Pls help.

![](https://z3.ax1x.com/2021/09/24/4DifOO.png)
![](https://z3.ax1x.com/2021/09/24/4Di4mD.png)

**Comment by loppefaaret** [comment]
do you see any WARNING in your `debug.log` file, located in your users folder `~/.chia/mainnet/log/` from the `harvester` about more than 5 seconds spend getting proofs from plot files ? are your full_node loosing sync at times ?

**Comment by BrandtH22** [comment]
@squallsdjl  , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
